packages/unity-interface/BrowserInterface.ts

```diff
@@ -106,6 +106,7 @@ import { ensureRealmAdapterPromise, getFetchContentUrlPrefixFromRealmAdapter } f
 import { setWorldLoadingRadius } from 'shared/scene-loader/actions'
 import { rendererSignalSceneReady } from 'shared/world/actions'
 import { requestMediaDevice } from '../shared/voiceChat/sagas'
+import { renderingActivated, renderingDectivated } from '../shared/loadingScreen/types'
```

Comment:
> DeactivateRenderingACK and ActivateRenderingACK control events dispatch actions for the loading screen. Since loading screen logic will be moved to renderer, we can set up these control events for deprecation as well
It is set for deprecetation already as a whole file: everything that is moved to a separate directory will be removed so I didn't mark each individual method
```
/** All members declared here are used for Loading screen reporting only, and thus are deprecated, and will be removed */
import { action } from 'typesafe-actions'

export const RENDERING_ACTIVATED = '[RENDERER] Camera activated'
export const renderingActivated = () => action(RENDERING_ACTIVATED)

export const RENDERING_DEACTIVATED = '[RENDERER] Camera deactivated'
export const renderingDectivated = () => action(RENDERING_DEACTIVATED)

export const RENDERING_FOREGROUND = '[RENDERER] Foreground'
export const renderingInForeground = () => action(RENDERING_FOREGROUND)

export const RENDERING_BACKGROUND = '[RENDERER] Background'
export const renderingInBackground = () => action(RENDERING_BACKGROUND)
```

Reply:
With everythign else I agree

Reply:
I also didn't annotate implementations of interfaces as it should be clear enough that they are deprecated as well